docs/csharp/programming-guide/concepts/expression-trees/debugview-syntax.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Syntax used by DebugView property (C#)
+description: Describes the special syntax used by the DebugView property to produce a string representation of expression trees
+author: zspitz
+ms.author: wiwagn
+ms.date: 05/22/2019
+ms.topic: reference
+helpviewer_keywords: expression-trees, debugview
+---
+
+# `DebugView` syntax
+
+The `DebugView` property (available only when debugging) provides a string rendering of expression trees. Most of the syntax is fairly straightforward to understand; the special cases are described in the following sections.
+
+Each example is followed by a block comment, containing the `DebugView`. 
+
+## ParameterExpression 
+
+<xref:System.Linq.Expressions.ParameterExpression> variable names are displayed with a `$` symbol at the beginning.
```

Comment:
```suggestion
<xref:System.Linq.Expressions.ParameterExpression?displayProperty=nameWithType> variable names are displayed with a `$` symbol at the beginning.
```

At least for the first xref, we should include the "displayProperty=nameWithType" parameter so that the fully qualified namespace is displayed. It wouldn't hurt to make this change on the first occurrence of each type. (that's one per paragraph). Otherwise, readers would only see the class name with no namespace info to help them find more about the related classes.

Reply:
Fixed.